pkg/apis/pipeline/v1beta1/pipeline_validation.go

```diff
@@ -247,6 +247,10 @@ func validateExecutionStatusVariablesExpressions(expressions []string, ptNames s
 		for _, expression := range expressions {
 			// check if it contains context variable accessing execution status - $(tasks.taskname.status)
 			if containsExecutionStatusRef(expression) {
+				// its a reference to aggregate status of dag tasks - $(tasks.status)
+				if expression == PipelineTasksAggregateStatus {
+					continue
+				}
```

Comment:
an alternative to updating the comment in line 248 to include `$(tasks.status)` could be moving this block before it

Reply:
done 👍